src/index.html

```diff
@@ -11,7 +11,142 @@
+            maxlength="30"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+         Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+         How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form_field">
+         Full date of birth:
+          <input
+            type="date"
+            name="birth"
+          >
+        </label>
+        <label class="form_field">
+         I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend class="content">Registration</legend>
+        <label class="form_field">
+         E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form_field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend class="content">An interesting fact about you!</legend>
+        <div class="form_field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="Y"
+            >
+          Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="N"
+            >
+          No
+          </label>
+        </div>
+        <label class="form_field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="form_field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="bedtime"
+            step="1"
+          >
+        </label>
+        <label class="form_field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form_field">
+          How do you rate our work?
+          <input type="range" name="rate">
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend class="content">Additional info:</legend>
+         Comments:
+        <textarea
+          name="comment"
+          cols="30"
+          rows="3"
+        >
+        </textarea>
```

Comment:
```suggestion
        ></textarea>
```
to prevent this 
<img width="407" alt="image" src="https://user-images.githubusercontent.com/53611910/201491341-65c38137-902d-4a81-a8ed-51b0eeb7c517.png">